src/component/ArticleList.jsx

```diff
@@ -0,0 +1,25 @@
+import {useState, useEffect} from "react";
+import ArticleCard from "./ArticleCard"
+import { getAllArticles } from "../api";
+
+const ArticleList = () => {
+
+ const [articles, setArticles] = useState([])
```

Comment:
There is no loading state here at the moment, on a slow connection I'd have no idea what is going on.